packages/wrangler/src/__tests__/configuration.pages.test.ts

```diff
@@ -0,0 +1,846 @@
+import { normalizeAndValidateConfig } from "../config/validation";
+import {
+	generateRawConfigForPages,
+	generateRawEnvConfigForPages,
+} from "./helpers/generate-wrangler-config";
+import type { RawConfig, RawEnvironment } from "../config";
+
+describe("normalizeAndValidateConfig()", () => {
```

Comment:
are we OK with these tests living in a separate file? it's rather unconventional to test the same fn over 2 separate test suites, but:
-`configuration.test.ts` is already 5k lines long so had mixed feelings abt adding more to it.
- this made it easy to write the tests
- this way we keep Pages specific tests in one place. Granted we could achieve that via a dedicated `describe` block inside `configuration.test.ts`

I have no strong opinions, but if anyone else does, happy to readjust.